packages/forms/resources/lang/nl/components.php

```diff
@@ -211,7 +211,7 @@
 
         'loading_message' => 'Laden...',
 
-        'max_items_message' => 'Er kunnen maar 3 geselecteerd worden.',
+        'max_items_message' => 'Er kunnen maar :count item(s) geselecteerd worden.',
```

Comment:
https://laravel.com/docs/9.x/localization#pluralization

```suggestion
        'max_items_message' => 'Er kunnen maar :count item geselecteerd worden.|Er kunnen maar :count items geselecteerd worden.',
```

Reply:
I don't think this is possible right now, since the maxItemsMessage is defined in the setup(). At this point we are still unsure how many items there might be. Or am I wrong?

Also we need to rewrite this part to use trans_choice.

Reply:
We actually have written a `trans_choice()` implementation in JavaScript, that's when this string gets evaluated... the number of max items never touches PHP

Reply:
Since the English translation didn't have "items", I left it out in the Dutch translation as well.